### ISSUE TITLE
Deprecation warning caused by old version of ember-cli-htmlbars

### ISSUE THREAD

**Issue by leizhao4** [zeppelin/ember-click-outside#3]
Warning message:
```
DEPRECATION: Overriding init without calling this._super is deprecated.
Please call `this._super.init && this._super.init.apply(this, arguments);`
addon: `ember-cli-htmlbars`
```

```
$ npm list ember-cli-htmlbars
├── ember-cli-htmlbars@1.0.8 
└─┬ ember-click-outside@0.0.3
  └── ember-cli-htmlbars@0.7.9 
```

Can you please update ember-cli-htmlbars to 1.0.x? Thanks!

**Comment by zeppelin** [comment]
Sure, thanks for reporting!

**Comment by niamleeson** [comment]
Hello,

The ember-cli-htmlbars version should be updated to 1.0.8 or above. The most recent release is 1.1.0. The `this._super` deprecation warning is fixed on 1.0.8. Could you please update it to 1.0.8?

Thank you! [Fix Page for HTMLBars](https://github.com/ember-cli/ember-cli-htmlbars/issues/77)
